### PR TITLE
Remove Plone < 5.2 and Python < 3 code.

### PR DESCRIPTION
**Pull request by ericof**
Fixes: #1140

### CODE REVIEW

**Comment by ericof** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]
LGTM! Maybe a second quick look/review would be good. Then we should merge and release asap to avoid merge conflicts.

File: src/plone/restapi/services/content/add.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
 from Acquisition import aq_base
 from Acquisition.interfaces import IAcquirer
+from plone.app.multilingual.interfaces import IPloneAppMultilingualInstalled
```
Comment (ale-rt) [inline src/plone/restapi/services/content/add.py:3]: I think the `plone.app.multilingual` import should be still in a try/except ImportError block, but maybe I am wrong.

File: src/plone/restapi/tests/test_documentation.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
 from base64 import b64encode
 from datetime import datetime
 from unittest.mock import patch
-from pkg_resources import parse_version
 from pkg_resources import resource_filename
 from plone import api
 from plone.app.discussion.interfaces import IConversation
 from plone.app.discussion.interfaces import IDiscussionSettings
 from plone.app.discussion.interfaces import IReplies
+from plone.app.multilingual.interfaces import ITranslationManager
```
Comment (ale-rt) [inline src/plone/restapi/tests/test_documentation.py:9]: I think the `plone.app.multilingual` import should be still in a try/except ImportError block, but maybe I am wrong.

File: src/plone/restapi/services/content/add.py (hunk 1 above)
Comment (erral) [reply src/plone/restapi/services/content/add.py:3]: No, plone.app.multilingual is included by default in Plone 5.2 and AFAIK there is no need to do a try: except: import.

**Review by ale-rt — APPROVED** [review]
LGTM

File: src/plone/restapi/services/content/add.py (hunk 1 above)
Comment (ale-rt) [reply src/plone/restapi/services/content/add.py:3]: Exactly: https://github.com/plone/Products.CMFPlone/blob/master/setup.py#L59

**Comment by tisto** [comment]
@ale-rt thanks for the swift review! :)

**Pull request merged**